src/pages/Register.ts

```diff
@@ -29,6 +30,7 @@ const constraints = (partOfOrg) => {
 const Register = () => {
   const user: TerraUser = getTerraUser();
   const profile: TerraUserProfile = authStore.get().profile;
```

Comment:
While you are in here, do you think you can remove this `TerraUserProfile` object, it cannot have any values in it as this gets populated after completing registration. As it is now, it is very confusing looking.